lib/Autoloader.php

```diff
@@ -58,6 +63,51 @@ public static function removePackage(Package $package)
 	}
 
 	// }}}
+	// {{{ public static function addFinalPackage()
+
+	/**
+	 * Sets an autoloader package as the last package to check against
+	 *
+	 * @param Package $package the autoloader package to add.
+	 *
+	 * @return void
+	 */
+	public static function setFinalPackage(Package $package)
```

Comment:
Recommend this method be renamed `addPackageAsFinal()` and make it update `self::$packages` as well as setting the `self::$final_package`